lnd.go

```diff
@@ -253,6 +253,12 @@ func Main(cfg *Config, lisCfg ListenerCfg, shutdownChan <-chan struct{}) error {
 	ctx, cancel := context.WithCancel(ctx)
 	defer cancel()
 
+	// Initialize etcd if LND compiled with etcd support and etcd is
+	// configured as remote backend.
+	if err := cfg.DB.InitEtcd(ctx, cfg.localDatabaseDir()); err != nil {
```

Comment:
Perhaps this instead can be hidden in `GetBackends`?

Reply:
Please see my other comment.